utils/python/CIME/case_setup.py

```diff
@@ -84,7 +84,7 @@ def case_setup(caseroot, clean=False, test_mode=False, reset=False):
 
     cimeroot = os.environ["CIMEROOT"]
 
-    case = Case()
+    case = Case(caseroot)
```

Comment:
Actually, this did not work unless I passed caseroot into the Case object for create_clone. It still does not work when I put in the original change. I don't understand why. I have checked that you really are in the caseroot directory but the queries in case_setup all seem to return None.